CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required (VERSION 2.8)
 project(OgreVideo)
 
-find_package(OGRE 1.11 REQUIRED COMPONENTS Bites)
+set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake")
+option(BUILD_DEMOS "Build demos (required Ogre Bites)" ON)
 
+find_package(OGRE3D)
```

Comment:
remove the custom (copypasted) find script. This gets out of sync with upstream rapidly. Instead make sure that the upstream version is copied to a system wide location.